kai/state/cstate/execution.go

```diff
@@ -124,8 +126,16 @@ func (blockExec *BlockExecutor) ApplyBlock(state LatestBlockState, blockID types
 		return state, block.Height(), fmt.Errorf("commit failed for application: %v", err)
 	}
 	state.AppHash = appHash
+	subStart := time.Now()
 	blockExec.store.Save(state)
 
+	if metrics.Enabled {
+		saveStateTimer.UpdateSince(subStart)
```

Comment:
Should we need timer evaluation metrics or storage metrics only @lewtran @thang14 ?

Reply:
this is ok for benchmarking purpose